src/content/community/online/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Online communities
+description: A listing of the grant programs throughout the Ethereum ecosystem.
+sidebar: true
+lang: en
+---
+
+# Online communities {#online-communities}
+
+Hundreds of thousands of Ethereum enthusiasts gather in these online forums to share news, talk about recent developments, debate technical issues, and imagine the future.
+
+## Forums
+
+- [r/ethereum](https://www.reddit.com/r/ethereum/) - _all things Ethereum_
+- [r/ethfinance](https://www.reddit.com/r/ethfinance/) - _the financial side of Ethereum, including DeFi_
+- [r/ethdev](https://www.reddit.com/r/ethdev/) - _focused on Ethereum development_
+- [r/ethtrader](https://www.reddit.com/r/ethtrader/) - _trends & market analysis_
+- [r/EtherMining](https://www.reddit.com/r/EtherMining/) - _focused on securing the Ethereum Network (mining)_
+- [r/ethstaker](https://www.reddit.com/r/ethstaker/) - _welcome to all interested in staking on Ethereum_
+- [Fellowship of Ethereum Magicians](https://ethereum-magicians.org) - _community oriented around technical standards in Ethereum_
+- [Ethereum Stackexchange](https://ethereum.stackexchange.com) - _discussion and help for Ethereum developers_
+- [Ethereum Research](https://ethresear.ch) - _the most influential messageboard for cryptoeconomic research_
+
+## Chat rooms
+
+- <Icon name="discord" /> <a href="https://gitter.im/ethereum/home">Ethereum Gitter</a> - <i>chat room for the Ethereum GitHub repo</i>
+- <Icon name="discord" /> <a href="https://discord.gg/tzYmDmF">Ethereum Cat Herders</a> - <i>community oriented around offering project management support to Ethereum development</i>
+- <Icon name="discord" /> <a href="https://ethglobal.co/discord">Ethereum Hackers</a> - <i>Discord chat run by ETHGlobal: an online community for Ethereum hackers all over the world</i>
+- <Icon name="discord" /> <a href="https://discord.gg/5W5tVb3">CryptoDevs</a> - <i>Ethereum development focused Discord community</i>
+- <Icon name="discord" /> <a href="https://discord.io/ethstaker">EthStaker Discord</a> - <i>community oriented around offering project management support to Ethereum development</i>
+- <Icon name="discord" /> <a href="https://discord.gg/CetY6Y4">Ethereum.org website team</a> - <i>stop by and chat ethereum.org web development and design with the team and folks from the community</i>
+
+## YouTube and Twitter
+
+- [Ethereum Foundation]() - _Keep upto date with the latest from the Ethereum Foundation_
+- [@ethereum]() - _Offical account of the Ethereum Foundation_
+- [@ethdotorg]() - _The portal to Ethereum, built for our growing global community_
+- [List of influential Ethereum twitter accounts]()
+
+<Divider />
+
+## Explore other online Ethereum communities
+
+- [DAOs](/community/daos/)
```

Comment:
Is this going to end up at `http://localhost:8000/en/community/get-involved/#decentralized-autonomous-organizations-daos`?